Makefile

```diff
@@ -11,6 +11,11 @@ dev:
 dev-vagrant:
 	pipenv run python ./manage.py runserver [::]:$(DEV_PORT)
 
+.PHONY: livereload
+livereload:
+	pipenv run python ./manage.py livereload $(DEV_LISTEN_IP):$(DEV_PORT)
```

Comment:
I don't think this'll make it run on the correct port, don't you run this alongside the regular web server to refresh when any changes are made? If so, I think the default port and host it uses are probably fine

Reply:
Admittedly fuzzy on the details of livereload, but from the cli usage:
```sh
pipenv run python ./manage.py livereload -h
usage: manage.py livereload [-h] [--version] [-v {0,1,2,3}]
                            [--settings SETTINGS] [--pythonpath PYTHONPATH]
                            [--traceback] [--no-color] [-l LIVEPORT]
                            [addrport]

Runs the development server with livereload enabled.

positional arguments:
  addrport              host and optional port the django server should listen
                        on (default: 127.0.0.1:8000)
```
it's the Django port specified, not the livereload server port (at 35729).